scripts/codestyle/run_uncrustify.sh

```diff
@@ -1,103 +1,68 @@
 #!/bin/bash
-if [ $# -gt 3 ]; then
-echo "Too many parameters specified"
-	exit 1
-fi
 
-VERBOSE=false
-HASHES=()
+TEMPDIR="out"
+UNCRUSTIFY_BASE=$(dirname $0)
+
+verbose="true"
+
+while getopts :qh opt; do
+	case $opt in
+	q)
+		verbose=""
+		;;
+	h)
+		echo "run_uncrustify.sh [options] [<git diff args>]"
+		echo ""
+		echo "Check codestyle of a change. Change generated by bypassing"
+		echo "diff-generating arguments to git-diff"
```

Comment:
> The change generated by passing
A verb is missing. `The change IS generated ...`

> No args means changes in worktree.
I still don't get it. What are other options besides using changes in the worktree? (BTW the "the" article in front of "worktree").
I though these args are supposed to alter the diff output, like the number of context lines, etc. What else could they affect?